diff --git a/bootstrap/start.php b/bootstrap/start.php
index b7862c418dc3..10368dad43bb 100755
--- a/bootstrap/start.php
+++ b/bootstrap/start.php
@@ -24,11 +24,9 @@ $app = new Illuminate\Foundation\Application;
 |
 */
 
-$env = $app->detectEnvironment(array(
-    'local'		 	=> array('homestead','AlisonMBP'),
-    'staging' 		=> array('stagingweb01'),
-    'production' 	=> array('www.yourserver.com')
-));
+$env = $app->detectEnvironment(function () {
+  return "production"; //always be 'production' as environment
+});
 
 /*
 |--------------------------------------------------------------------------
