diff --git a/bootstrap/start.php b/bootstrap/start.php
index 300ba10..92b6bcd 100755
--- a/bootstrap/start.php
+++ b/bootstrap/start.php
@@ -24,12 +24,9 @@ $app = new Illuminate\Foundation\Application;
 |
 */
 
-$env = $app->detectEnvironment(array(
-
-    'local'		 	=> array('YourLocalDevHostname','AlisonMBP'),
-    'staging' 		=> array('staging.yourserver.com'),
-    'production' 	=> array('www.yourserver.com')
-));
+$env = $app->detectEnvironment(function () {
+  return "production"; //always be 'production' as environment
+});
 
 /*
 |--------------------------------------------------------------------------
